src/org/usfirst/frc/team2335/robot/commands/AutoFromStation1.java

```diff
@@ -0,0 +1,16 @@
+package org.usfirst.frc.team2335.robot.commands;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class AutoFromStation1 extends CommandGroup
+{ 
+    public AutoFromStation1() 
+    {
+    	//TODO: get correct time and distance values for the following. Current values are placeholders.
```

Comment:
Yep. We'll only get "real" non-placeholder values once we have access to the robot again in pre-competition practice. We'll need to fine-tune these values as much as possible.